portal/server/routes/utilisation-report-service/utilisation-report-upload/index.js

```diff
@@ -9,18 +9,24 @@ const {
 } = require('../../../controllers/utilisation-report-service');
 const { validateRole, validateToken, virusScanUpload } = require('../../middleware');
 const { utilisationReportMulterFilter, formatBytes } = require('../../../utils/multer-filter.utils');
-const { ROLES } = require('../../../constants');
+const { ROLES, LANDING_PAGES } = require('../../../constants');
 
 const { MAX_UTILISATION_REPORT_FILE_SIZE_BYTES } = process.env;
 
 const router = express.Router();
 
-const upload = multer({ limits: { fileSize: +MAX_UTILISATION_REPORT_FILE_SIZE_BYTES }, fileFilter: utilisationReportMulterFilter }).single('utilisation-report-file-upload');
+const upload = multer({ limits: { fileSize: +MAX_UTILISATION_REPORT_FILE_SIZE_BYTES }, fileFilter: utilisationReportMulterFilter }).single(
+  'utilisation-report-file-upload',
+);
 
-router.get('/utilisation-report-upload', [validateToken, validateRole({ role: [ROLES.PAYMENT_REPORT_OFFICER] })], (req, res) => getUtilisationReportUpload(req, res));
+router.get(
+  LANDING_PAGES.UTILISATION_REPORT_UPLOAD,
```

Comment:
Same comment here. It doesn't feel right to be referring to "LANDING_PAGES" in this context.